.eslintrc

```diff
@@ -10,7 +10,7 @@
     "import"
   ],
   "rules": {
-    "comma-dangle": [2, "always-multiline"],
+    "comma-dangle": [2, "only-multiline"],
```

Comment:
@minsooshin Let's talk about trailing commas in JS object.

Reply:
the reason why trailing comma is useful is it is easy to see the diff when the code change
if you don't have trailing comma, and add more property, diff says two lines has been changed, but logically one property has been added. If it has trailing comma, diff only says one line added since we don't add comma and line. So, it helps on diff to have more actual changes.
In this reason, I prefer to have trailing comma on any object. So, I added this rule in eslint.

Reply:
Checked from eslint documentation, the disadvantage of trailing comma is ES5 and old version browsers(like IE8...) don't support it. 
I think this is more like a convention problem. If most of react enterprise products are using the trailing comma for the object definition, I agree to use this way.

Reply:
I would like to let you know one thing, we are compile these es6 components to es5 components and it removes the trailing comma while the compiling. You can find this from our files in `dist` directory. So, it is not an issue at all. For, not compiled source, webpack is handling these thing by `babel-loader`, so that old browsers are not a problem.

Reply:
OK. Then I agree to keep the trailing comma.